src/rpc/blockchain.h

```diff
@@ -41,8 +41,10 @@ void RPCNotifyBlockChange(const CBlockIndex*);
 /** Block description to JSON */
 UniValue blockToJSON(const CBlock& block, const CBlockIndex* tip, const CBlockIndex* blockindex, bool txDetails = false) LOCKS_EXCLUDED(cs_main);
 
+typedef std::vector<CAmount> MempoolHistogramFeeRates;
+
 /** Mempool information to JSON */
-UniValue MempoolInfoToJSON(const CTxMemPool& pool);
+UniValue MempoolInfoToJSON(const CTxMemPool& pool, const std::optional<MempoolHistogramFeeRates> feeLimits);
```

Comment:
Need to `#include <optional>` in this header

Reply:
Yes, thank you.